src/views/territory-choice-general-partner-usual-residential-address.njk

```diff
@@ -1,24 +1,14 @@
 {% extends "layout.njk" %}
 
-{% block backLink %}
-  {% if props.data.generalPartner.data.forename %}
-    {% set hrefValue = props.currentUrl | replace(props.pageType, "add-general-partner-person") %}
-  {% elif props.data.generalPartner.data.legal_entity_name %}
-    {% set hrefValue = props.currentUrl | replace(props.pageType, "add-general-partner-legal-entity") %}
-  {% else %}
-     {% set hrefValue = props.previousUrl %}
-  {% endif %}
+{# Back Link #}
+{% if props.data.generalPartner.data.forename %}
+  {% set hrefValue = props.currentUrl | replace(props.pageType, "add-general-partner-person") %}
+{% else %}
+  {% set hrefValue = props.previousUrl %}
+{% endif %}
 
-  {{ govukBackLink({
```

Comment:
back link seems to work without re-defining the backLink component, just by setting hrefValue